doc/src/_static/default.css_t

```diff
@@ -290,10 +290,12 @@ p.admonition-title:after {
 }
 
 pre {
+    font-family: DejaVu Sans Mono, monospace;
+    font-size: 80%;
     padding: 5px;
     background-color: {{ theme_codebgcolor }};
     color: {{ theme_codetextcolor }};
-    line-height: 120%;
+    line-height: 100%;
```

Comment:
It does seem that this is needed for Chrome to avoid spaces in integrals.

Reply:
Which is unfortunate as the extra space makes the docstring examples a little more readable. In Firefox, with 120% they are still connected.

Reply:
I got your point I changed it to 100% because i thought with 100% it was looking nice . 
Now I had changed it to 120% .